From: Tianyu Chen <sweetyfish@deepin.org>
Date: Tue, 3 Feb 2026 15:26:16 +0800
Subject: conf: Remove format parameter from plist_from_memory() for libplist
 2.2.0 compatibility

The format parameter was added in libplist 2.3.0 and doesn't exist
in 2.2.0. Removing it ensures backward compatibility with the older
version.
---
 src/conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/conf.c b/src/conf.c
index 96d82c9..30708eb 100644
--- a/src/conf.c
+++ b/src/conf.c
@@ -417,7 +417,7 @@ int config_set_device_record(const char *udid, char* record_data, uint64_t recor
 	}
 
 	plist_t plist = NULL;
-	plist_from_memory(record_data, record_size, &plist, NULL);
+	plist_from_memory(record_data, record_size, &plist);
 
 	if (!plist || plist_get_node_type(plist) != PLIST_DICT) {
 		if (plist)
